From 085c65e91273bbeb3e2d70298c795a027406a62b Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Thu, 17 Jul 2014 17:11:06 +1000
Subject: [PATCH 03/16] metadata: clear error state once an error has been
 propagated

Otherwise following successful downloads would have been marked as failed too
---
 mythtv/libs/libmythmetadata/metadataimagedownload.cpp |    1 +
 1 file changed, 1 insertion(+)

diff --git a/mythtv/libs/libmythmetadata/metadataimagedownload.cpp b/mythtv/libs/libmythmetadata/metadataimagedownload.cpp
index b798e7a..711187b 100644
--- a/mythtv/libs/libmythmetadata/metadataimagedownload.cpp
+++ b/mythtv/libs/libmythmetadata/metadataimagedownload.cpp
@@ -330,6 +330,7 @@ void MetadataImageDownload::run()
         {
             QCoreApplication::postEvent(m_parent,
                     new ImageDLFailureEvent(lookup));
+            errored = false;
         }
         lookup->SetDownloads(downloaded);
         QCoreApplication::postEvent(m_parent, new ImageDLEvent(lookup));
-- 
1.7.10.2

